### PR TITLE
Update qhull to revision 5bbc756

### PR DESCRIPTION
**Pull request by dsogari**

### CODE REVIEW

**Comment by Alexpux** [comment]
I can't accept it as it breaks versioning of packages

File: mingw-w64-qhull-git/PKGBUILD [hunk 1]
```diff
@@ -18,50 +19,38 @@ depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs")
...
              "${MINGW_PACKAGE_PREFIX}-cmake")
-source=(#"${_realname}"::"git://gitorious.org/qhull/qhull.git"
-        "${_realname}"::"git+https://github.com/manctl/qhull.git"
-        "0001-MSYS2-Adopt-FHS-layout-in-cmake.patch"
-        "0002-MinGW-w64-64-bit-ptr_intT-fix.patch")
-sha256sums=('SKIP'
-            '347f7f0f48bc24ffcb2649c188ab6f39dfbe8972ee002857dce4fca74c088148'
-            '9aa307018c99e6270f2f7921a8a639ae26d7ce9900483fe70af7d7082f40e86f')
-
-pkgver() {
```
Comment (Alexpux) [inline mingw-w64-qhull-git/PKGBUILD]: don't remove pkgver function

Comment (dsogari) [inline mingw-w64-qhull-git/PKGBUILD]: I'm sorry. Will fix it

**Comment by dsogari** [comment]
Now I understand how git versioning of packages works :)

**Pull request merged**